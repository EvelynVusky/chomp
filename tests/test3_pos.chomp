/* operations */

int x = 5; 
int y = 10; 

int operations (int a, int b) {
    return a + b / 3 + (4 * 6);
}

int compare(int a, int b){
    if (a == b) {
        return 4;
    } else {
        return 0;
    }
    if (a >= b) {
        return 5;
    }
    if (a <= b) {
        return 1;
    }
    if (a < b) {
        return 4;
    }
    if (a != b) {
        return 10;
    }
}

int bool_compare(bool c, bool d){
    if (c || d) {
        return 1;
    }
    if (c && d){
        return 4;
    }
}

int call_func() {
    operations(x, y);
}

list char world = ['w', 'o', 'r', 'l', 'd'];
void printing() {
    print (['h', 'e', 'l', 'l', 'o']);
    print (world);
    println ([1, 2, 3, 4, 5]);
    println (1 + 1);
}