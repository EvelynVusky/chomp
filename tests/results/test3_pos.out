int x = 5;
int y = 10;
list char world = ['w', 'o', 'r', 'l', 'd'];

int operations(a, b)
{
return a + b / 3 + 4 * 6;
}

int compare(a, b)
{
if (a == b)
{
return 4;
}
else
{
return 0;
}
if (a >= b)
{
return 5;
}
if (a <= b)
{
return 1;
}
if (a < b)
{
return 4;
}
if (a != b)
{
return 10;
}
}

int bool_compare(c, d)
{
if (c || d)
{
return 1;
}
if (c && d)
{
return 4;
}
}

int call_func()
{
operations(x, y);
}

void printing()
{
print ['h', 'e', 'l', 'l', 'o'];
print world;
println [1, 2, 3, 4, 5];
println 1 + 1;
}
